test/torchaudio_unittest/sox_effect/sox_effect_test.py

```diff
@@ -127,6 +128,33 @@ def test_apply_effects(self, args):
         assert sr == expected_sr
         self.assertEqual(found, expected)
 
+    @parameterized.expand(
+        load_params("sox_effect_test_args.json"),
+        name_func=lambda f, i, p: f'{f.__name__}_{i}_{p.args[0]["effects"][0][0]}',
+    )
+    def test_apply_effects_path(self, args):
```

Comment:
This is overkill for testing the `Pathlib.Path` compatibility. Can you just pick up one test case, so that the focus of the test is intuitively clear that it's about `Pathlib.Path`.

Reply:
@mthrok  changed the pathlib unit test by hard coding a specific test case argument